src/wwElement.vue

```diff
@@ -70,6 +70,9 @@ export default {
                 textDecoration: this.content.textDecoration,
                 textDecorationStyle: this.content.textDecorationStyle,
                 textDecorationColor: this.content.textDecorationColor,
+                overflow: this.content.nowrap ? 'hidden' : 'initial',
```

Comment:
no collission with the overflow property?

Reply:
No, the overflow property is applied to the ww-element, not the text inside.
So the ellipsis property does not work with the overflow element property

Reply:
Does it make sense to have two properties ? Should we set overflow to hidden automatically when ellipsis is set ?
Should ellipsis be a select with all text-overflow options ? 🤔

Reply:
https://www.loom.com/share/dfc3f23a61e543f6a8813470ee9352d9

Reply:
Merci pour les explications je comprends mieux pourquoi les autres options marchaient pas quand je testais sur mdn